nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -130,7 +131,7 @@ public DocumentModel createComment(DocumentModel docModel, DocumentModel comment
                     (Serializable) computeAncestorIds(session, docModel.getId()));
             DocumentModel comment = session.createDocument(commentModelToCreate);
             comment.detach(true);
-            notifyEvent(session, CommentEvents.COMMENT_ADDED, docModel, comment);
+            notifyEvent(session, comment, CommentEvents.COMMENT_ADDED);
```

Comment:
for the creation we can use the previous method but my idea was to be consistent with the others operation `Update` and `Removing` and i told my self even if we reload the document  if no cache exist it will not cost much

Reply:
Great though, but here we can leverage on fact we already have fetched the document.